Lib/test/test_tools/test_i18n.py

```diff
@@ -155,206 +155,6 @@ def test_POT_Creation_Date(self):
             # This will raise if the date format does not exactly match.
             datetime.strptime(creationDate, '%Y-%m-%d %H:%M%z')
 
-    def test_funcdocstring(self):
```

Comment:
Even if we're using `ast.get_docstring`, we still want to check that using the `--docstring` command-line option is correct or don't we?